modules/eslint-plugin/schematics/ng-add/index.ts

```diff
@@ -4,17 +4,79 @@ import type { Schema } from './schema';
 
 export default function addNgRxESLintPlugin(schema: Schema): Rule {
   return (host: Tree, context: SchematicContext) => {
-    const eslintConfigPath = '.eslintrc.json';
+    const jsonConfigPath = '.eslintrc.json';
+    const flatConfigPath = 'eslint.config.js';
     const docs = 'https://ngrx.io/guide/eslint-plugin';
 
-    const eslint = host.read(eslintConfigPath)?.toString('utf-8');
+    // If a flat config exists, update it
+    if (host.exists(flatConfigPath)) {
+      let content = host.read(flatConfigPath)?.toString('utf-8');
```

Comment:
This method is becoming lengthy, could you split it up into at least two methods, one for the flat config, the other for the JSON file?